### PR TITLE
feature/SM-6605: docs updates

### PR DESCRIPTION
**Pull request by ccurran01**
Documentation for get files error response updates

### CODE REVIEW

File: docs/api-documentation/V1.25.9/index.md [hunk 1]
```diff
@@ -2783,6 +2783,17 @@ Future releases will include new features relating to Comments. These include:
 This section lists any significant changes made to this document (and by extension, the API interfaces themselves) introduced by each recent and upcoming future release.
 {: .govuk-body}
 
+Version 1.25.9 - Stable (21/01/2021):
```
Comment (pwright08) [inline docs/api-documentation/V1.25.9/index.md:2786]: Version and changes now moved to separate file as part of https://github.com/departmentfortransport/street-manager-docs/pull/649

File: docs/api-documentation/V1.25.9/index.md [hunk 2]
```diff
@@ -2783,6 +2783,17 @@ Future releases will include new features relating to Comments. These include:
...
+{: .govuk-heading-s}
+
+Updated Works API with the following changes:
+{: .govuk-body}
+<ol class="govuk-list govuk-list--bullet">
+  <li>
+    SM-6605: Updated the format of <code>GET ​/files​/{fileId}</code> error response body to be in line with error responses throughout Street Manager. An error response body will now contain an object where previously it contained only the string title of the error code. No further error scenarios have been introduced as part of this change.
```
Comment (pwright08) [inline docs/api-documentation/V1.25.9/index.md:2793]: Wondering if the change in error response body format here is enough to call this out as a breaking change?

Comment (ccurran01) [reply docs/api-documentation/V1.25.9/index.md]: I can, it is technically breaking

File: docs/api-documentation/V1.25.9/changelog.md [hunk 3]
```diff
@@ -11,6 +11,14 @@ This section lists any significant changes made to this document (and by extensi
 Version 1.25.9 - Stable (21/01/2021):
 {: .govuk-heading-s}
 
+Updated Works API with the following changes:
+{: .govuk-body}
+<ol class="govuk-list govuk-list--bullet">
+  <li>
+    BREAKING CHANGE SM-6605: Updated the format of <code>GET ​/files​/{fileId}</code> error response body to be in line with error responses throughout Street Manager. An error response body will now contain an object where previously it contained only the string title of the error code. No further error scenarios have been introduced as part of this change.
```
Comment (pwright08) [inline docs/api-documentation/V1.25.9/changelog.md:18]: Would also add the breaking change tag in the v2 docs since it should be stable now too

Comment (ccurran01) [reply docs/api-documentation/V1.25.9/changelog.md:18]: fair point

File: docs/api-documentation/V1.25.9/index.md [hunk 4]
```diff
@@ -2776,1193 +2776,3 @@ Future releases will include new features relating to Comments. These include:
   and time the comment was marked as read</li>
   <li>New filter options on comment list pages and exports, including filtering by comment date and topic</li>
 </ol>
-
-## Versions and Changes
```
Comment (ccurran01) [inline docs/api-documentation/V1.25.9/index.md:2780]: A PR that was out of date with master was merged so this was added back again

File: docs/api-documentation/V1.25.9/index.md [hunk 5]
```diff
@@ -2776,1193 +2776,3 @@ Future releases will include new features relating to Comments. These include:
...
-
-This section lists any significant changes made to this document (and by extension, the API interfaces themselves) introduced by each recent and upcoming future release.
-{: .govuk-body}
-
-Version 1.25.9 - Stable (25/01/2021):
-{: .govuk-heading-s}
-
-Updated Work API with the following changes:
-{: .govuk-body}
-<ol class="govuk-list govuk-list--bullet">
```
Comment (caoimheb) [inline docs/api-documentation/V1.25.9/index.md:2791]: This update for SM-6114 will need added to the new changelog

And the same will need done for the v2.10.4 changelog

File: docs/api-documentation/V1.25.9/changelog.md [hunk 6]
```diff
@@ -11,6 +11,17 @@ This section lists any significant changes made to this document (and by extensi
 Version 1.25.9 - Stable (21/01/2021):
 {: .govuk-heading-s}
 
+Updated Works API with the following changes:
+{: .govuk-body}
+<ol class="govuk-list govuk-list--bullet">
+  <li>
```
Comment (caoimheb) [inline docs/api-documentation/V1.25.9/changelog.md:17]: In the previous update for SM-6114 there is the text "The following business rule change introduced on 25/01/2021 is included:"

Looks from the PR like we need to keep that format - https://github.com/departmentfortransport/street-manager-docs/pull/644

**Review by pwright08 — APPROVED** [review]

File: docs/api-documentation/V2.10.4/changelog.md [hunk 7]
```diff
@@ -23,6 +23,8 @@ Updated Work API with the following changes:
         SM-6114: Updated permit cancel logic to only allow active permits to be cancelled, making it consistent with the options available on the UI.
       </li>
     </ol>
+  <li>
```
Comment (caoimheb) [inline docs/api-documentation/V2.10.4/changelog.md:26]:
```suggestion
  </li>
  <li>
```

Comment (ccurran01) [reply docs/api-documentation/V2.10.4/changelog.md:27]: I think that should be that sorted now

**Review by caoimheb — APPROVED** [review]

**Pull request merged**